zeebe/engine/src/main/java/io/camunda/zeebe/engine/processing/tenant/TenantUpdateProcessor.java

```
@@ -111,11 +111,9 @@ private boolean tenantIdConflict(final TenantRecord record) {
 
   private void updateExistingTenant(
       final TenantRecord existingTenant, final TenantRecord updateRecord) {
-    if (!updateRecord.getTenantId().isEmpty()) {
-      existingTenant.setTenantId(updateRecord.getTenantId());
-    }
-    if (!updateRecord.getName().isEmpty()) {
-      existingTenant.setName(updateRecord.getName());
+    final var updatedName = updateRecord.getName();
+    if (!updatedName.isEmpty()) {
+      existingTenant.setName(updatedName);
     }
```

Can we throw an exception here if the tenant id is non-empty?